source/config/routes.rb

```diff
@@ -1,9 +1,18 @@
 Rails.application.routes.draw do
+  get 'admin/index'
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
   root 'sessions#index'
+  post '/' => 'sessions#create'
+  get 'sessions' => 'sessions#index'
```

Comment:
Thanks Jay!

`resources :sessions, only: [:index, :create, :destroy]` gave me this route:
```bash
bundle exec rake routes
#=> 
...
    session DELETE /sessions/:id(.:format) sessions#destroy
...
```

which led to using `%li=link_to "Logout", "/sessions/1", method: 'delete', action: 'destroy'`
i.e. the `1` is hard-coded to fit the route. 

So, I tried `resource :sessions, only: [:index, :create, :destroy]`
and this led to these routes
```bash
   sessions POST   /sessions(.:format)    sessions#create
            DELETE /sessions(.:format)    sessions#destroy
``` 
but no GET route.

So, I found that using this combination:
```ruby
  resources :sessions, only: [:index]
  resource :sessions, only: [:index, :create, :destroy]
```
led to all three of the routes the app used:
```bash
   sessions GET    /sessions(.:format)    sessions#index
            POST   /sessions(.:format)    sessions#create
            DELETE /sessions(.:format)    sessions#destroy
```

I refered to http://guides.rubyonrails.org/routing.html in this investigation.